pkg/vault/vault.go

```diff
@@ -172,22 +173,22 @@ func (v *Vault) validateExternalConfiguration() error {
 	}
 
 	if v.ServiceAccountName == "" {
-		err = errors.New("service account name cannot be empty")
+		err = errors.New("external vault service account name cannot be empty")
```

Comment:
if you want to change the messages, I think "service account name for external vault instance cannot be empty" would be more appropriate. 
Same applies for the other messages.

Reply:
oh my word

Reply:
?